core/src/main/java/com/linecorp/armeria/common/logging/DefaultRequestLog.java

```diff
@@ -622,6 +623,12 @@ public String toString() {
 
     @Override
     public String toStringRequestOnly() {
+        return toStringRequestOnly(Function.identity(), Function.identity());
+    }
+
+    @Override
+    public String toStringRequestOnly(Function<HttpHeaders, HttpHeaders> headersSanitizer,
+                                      Function<Object, Object> contentSanitizer) {
```

Comment:
Should the sanitizers be `Function<? super HttpHeaders, ?>` and `Function<Object, ?>` instead?